src/xdocs/config.xml

```diff
@@ -952,30 +957,18 @@
 
     </section>
 
-    <section name="XML Structure">
+    <section name="XML Configuration Structure">
 
-      <b>Configuration XML Document</b>
       <p>
-        The following DTD for a configuration XML document specifies the
-        hierarchy of modules and their properties:
+        DTD is used to validate
+        configuration XML document which specifies the
+        hierarchy of modules and their properties.
       </p>
 
-      <source>
-&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
-
-&lt;!ELEMENT module (module|property)*&gt;
-&lt;!ATTLIST module name NMTOKEN #REQUIRED&gt;
-
-&lt;!ELEMENT property EMPTY&gt;
-&lt;!ATTLIST property
-        name NMTOKEN #REQUIRED
-        value CDATA #REQUIRED
-&gt;
-      </source>
-
       <p>
         Checkstyle validates a configuration XML document when it loads
```

Comment:
`validates a configuration XML document` => `validates the configuration XML document`
`XML document` => `XML document structure`.

Reply:
done

Reply:
@romani Same here.

Reply:
strange. I did it. 
done.
verified after push to remote